lib/docs/stories/introduction/pam.stories.mdx

```diff
@@ -4,9 +4,9 @@ import { Meta } from "@storybook/addon-docs/blocks";
 
 # PubNub Access Manager
 
-PubNub Access Manager (PAM) allows you to enforce secure controls for client access to resources
+PubNub Access Manager (PAM) allows you to enforce security controls for client access to resources
```

Comment:
```suggestion
Access Manager allows you to enforce security controls for client access to resources
```

Reply:
Ok, I renamed it to Access Manager throughout the doc.